### PR TITLE
Add hard liveliness check

### PR DESCRIPTION
**Pull request by pablogs9**

### CODE REVIEW

File: docs/client.rst [hunk 1]
```diff
@@ -290,6 +290,14 @@ By means of these flags, the user can change the default value of all the parame
...
         - :code:`<number>`
         - :code:`10`
+    *   - :code:`UCLIENT_HARD_LIVELINESS_CHECK`
+        - Enables Micro XRCE-DDS Client hard liveliness check.
+        - :code:`<bool>`
+        - :code:`OFF`
+    *   - :code:`UCLIENT_HARD_LIVELINESS_CHECK_TIMEOUT`
+        - Sets Micro XRCE-DDS Client hard liveliness check timeout in milliseconds.
+        - :code:`<number>`
+        - :code:`10000`
```
Comment (Acuadros95) [inline docs/client.rst:300]: Can we add here a `max value` tag?

Comment (pablogs9) [reply docs/client.rst:300]: Maybe in the description.

File: docs/client.rst [hunk 2]
```diff
@@ -290,6 +290,14 @@ By means of these flags, the user can change the default value of all the parame
         - This value corresponds to the *Max number data buffers stored in shared memory*.
         - :code:`<number>`
         - :code:`10`
+    *   - :code:`UCLIENT_HARD_LIVELINESS_CHECK`
+        - Enables Micro XRCE-DDS Client hard liveliness check.
+        - :code:`<bool>`
+        - :code:`OFF`
+    *   - :code:`UCLIENT_HARD_LIVELINESS_CHECK_TIMEOUT`
+        - Sets Micro XRCE-DDS Client hard liveliness check timeout in milliseconds.
```
Comment (Acuadros95) [inline docs/client.rst:298]:
```suggestion
        - Sets Micro XRCE-DDS Client hard liveliness check timeout in milliseconds, max value: "999999".
```

Comment (pablogs9) [reply docs/client.rst]: Addressed 1aed9a2c3476badc41944f8d0ec7ad1de67ebfe4

**Review by Acuadros95 — APPROVED** [review]

**Pull request merged**